server/services/placeOnReportService.ts

```diff
@@ -247,6 +247,7 @@ export default class PlaceOnReportService {
       handoverDeadline: draftAdjudication.incidentDetails.handoverDeadline,
       statementPresent: !!draftAdjudication.incidentStatement,
       statementComplete: draftAdjudication.incidentStatement?.completed || false,
+      offenceDetailsComplete: draftAdjudication.offenceDetails?.length > 0,
```

Comment:
I'm under the impression that the offence details array will either be not present at all, or have at least 1 element (on the draft adjudication)

Reply:
Yep - exactly - and I assume having 1 or more means it is complete?!